Dockerfile

```diff
@@ -9,3 +9,6 @@ RUN cd /go/src/github.com/openshift/cluster-openshift-apiserver-operator && go b
 
 FROM centos:7
 COPY --from=0 /go/src/github.com/openshift/cluster-openshift-apiserver-operator/cluster-openshift-apiserver-operator /usr/bin/cluster-openshift-apiserver-operator
+
+COPY manifests/install /manifests
```

Comment:
@deads2k would we need to split the single operator.yaml file under the install folder into one file per resource? I'm assuming we'd need to name each file accordingly to ensure some resources are created before others?

Reply:
> @deads2k would we need to split the single operator.yaml file under the install folder into one file per resource? I'm assuming we'd need to name each file accordingly to ensure some resources are created before others?

split whatever is needed.  Make our `manifest` dir be in `manifest`.  Move whatever is needed.

Reply:
Also, ordering is a dangerous precedent, but if it is required, yes.